### PR TITLE
[minor/bug] fix to build windows agent

### PR DESCRIPTION
**Pull request by yvesago**
Hello,

here is my first pull request for mig ;-)

It contains some simple fixes to allow building a windows client with the new agentcontext feature and fix the old bug in logging_windows.
Building module memory still fail.

### CODE REVIEW

File: logging_windows.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	"os"
 	"sync"
 
-	"code.google.com/p/winsvc/eventlog"
+    "golang.org/x/sys/windows/svc/eventlog"
```
Comment (ameihm0912) [inline logging_windows.go]: it looks like we have some indenting issues here -- could you run gofmt on your files changed, just to make sure we stay consistent with indenting and so on?

**Comment by ameihm0912** [comment]
@yvesago thanks for grabbing this one. It looks good to me, if we could just fix up any formatting issues via gofmt.

**Comment by yvesago** [comment]
This one should be better (if fixup works as I hope)

**Comment by ameihm0912** [comment]
Looks good, thanks!

**Pull request merged**